babel.config.js

```diff
@@ -468,6 +469,13 @@ function pluginPolyfillsOldNode({ template, types: t }) {
         process.allowedNodeEnvironmentFlags || require("node-environment-flags")
       `,
     },
+    {
+      name: "Object.hasOwn",
```

Comment:
Can you add a comment here that `Object.hasOwn` was introduced in Node.js 16.9.0?